3dtools/tools.py

```diff
@@ -22,9 +22,9 @@
 
 # special constant for all the function
 RATIO = 781/85.4  # pixel/mm = 9.145

```

Comment:
Shouldn't this be recalculated for each image? Using the image dimension in pixel and user given height or width